templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver {{ driver.username }}</h1>
+  <p>Are you sure you want to delete {{ driver.first_name }} {{ driver.last_name }} from database?</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, I'm sure" class="btn btn-danger">
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist